script.js

```diff
@@ -0,0 +1,6 @@
+"use strict";
+
+const lielement = document.createElement("li");
+lielement.textContent = "これです";
+const ulelement = document.querySelector("ul");
+ulelement.appendChild(lielement);
```

Comment:
I think the camel case is the way to go here. Take a close look at the code your teacher wrote.
If you want, you can refer to it.

https://designsupply-web.com/media/development/4052/

Reply:
I overlooked the capitalization part.
Thank you very much. I will refer to it.